bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/SetupListener.java

```diff
@@ -56,7 +56,7 @@ private boolean compatible(Alloc pred, Alloc succ) {
 		return 
 			pred != null &&
 			pred.getObs().getInstrumentString().equals(succ.getObs().getInstrumentString()) &&
-			pred.getObs().getCoords(pred.getMiddlePoint()).equals(succ.getMiddlePoint());
```

Comment:
I think maybe what we want here is:

```java
pred.getObs().getCoords(pred.getEnd()).equals(succ.getObs().getCoords(pred.getEnd()))
```

That fixes the bug where we are comparing two different types (`WorldCoords` vs. `Long`) and uses the same time, in this case `pred.getEnd()`, for the two observations.